config/registry/aws/modules/cloudfront-distribution.yaml

```diff
@@ -0,0 +1,60 @@
+halt: false
+environment_module: false
+terraform_module: aws-cloudfront-distribution
+inputs:
+  - name: env_name
+    user_facing: false
+    description: Opta Environment name
+    default: None
+  - name: layer_name
+    user_facing: false
+    description: Opta Layer name
+    default: None
+  - name: module_name
+    user_facing: false
+    description: Opta Module name
+    default: None
+  - name: s3_log_bucket_name
+    user_facing: false
+    description: Name of the access log bucket
+    default: null
+  - name: bucket_name
```

Comment:
I think bucket_name and origin_access_identity_path should also be user_facing and not required. This way someone can use a bucket that they created outside of opta.